src/shadowbox/server/api.yml

```diff
@@ -418,6 +418,35 @@ paths:
               examples:
                 '0':
                   value: '{"bytesTransferredByUserId":{"1":1008040941,"2":5958113497,"3":752221577}}'
+  /metrics/tunnel-time-location
```

Comment:
Can this be `/metrics/tunnel-time`? I thought you mention there was another one per access key, but I don't see it. How are we going to report by key?

Did Sander have an opinion on this?

Reply:
We will know better about the API once we start using it for real. So perhaps we should pause this and focus on the manager side to better understand the usage.

For instance, we need tunnel time per access key too. Will the manager be forced to issue 3 queries?
Perhaps it will be best to issue one query, maybe under `/metrics/usage`, that returns all the usage metrics we need. That will simplify the Manager. It will also let us deprecate the bad `transfer` endpoint. In the future we can just add more to the usage endpoint, instead of keeping adding new endpoints per metric.